### DISCUSSION TITLE
Frigate LXC storage configuration

### DISCUSSION THREAD

**Discussion by maxfun73v2** [tteck/Proxmox#2924] [Help Needed] [answered]
Hi, I installed the Frigate LXC on my Proxmox Instance but I cant' find how to change the recordings/clips folder.
Thanks.

**Comment by nautilus7** [comment]
I installed frigate lxc yesterday as well. I haven't found any way to do this during the lxc creation. I guess you need to add a mountpoint from the lxc resources option (mount an existing folder from the host to the lxc), or in case of a privileged lxc you can directly mount an nfs share from inside the lxc.

My question in the 2nd case (mount nfs inside lxc), do I need to stop the frigate app before mounting? How do I do that (just frigate, not the whole lxc)?

Nevertheless, it would be great to add an option to do it during the installation process.

**Reply by maxfun73v2** [reply]
I run a privileged LXC container with a mount of a network share, I would like to place recordings in this share but I don't know how to configure it.

I can add the mountpoint to the LXC but it's the same.
I don't think you have to stop Frigate before the mount of the share.

**Reply by nautilus7** [reply, marked as answer]
Frigate uses /media/frigate for its recordings, clips, etc, so I mounted my share in that folder, which already exists in the lxc.
You first need to install `nfs-common` in the lxc and then add a line in the `/etc/fstab` to make the mount permanently:

 `192.168.1.1:/data /media/frigate nfs defaults 0 0`

There are many [instructions](https://theorangeone.net/posts/mount-nfs-inside-lxc/) online.

**Reply by wildchai** [reply]
can this be done to mount a directory to that folder as well? I do not have an NFS share.

**Comment by maxfun73v2** [comment]
> Frigate uses /media/frigate for its recordings, clips, etc, so I mounted my share in that folder, which already exists in the lxc.

Thank you so much for your help, I didn't think to mount the existing folder to my share.

**Reply by nautilus7** [reply]
Maybe there is an option in the frigate config to use a different folder for recordings, I can't remember. In this case you could mount the share wherever you like and adjust your config.
